app/plugin_api.go

```diff
@@ -81,6 +81,11 @@ func (api *PluginAPI) GetConfig() *model.Config {
 	return api.app.GetSanitizedConfig()
 }
 
+// GetUnsanitizedConfig fetches the currently persisted config without removing sensible configuration
+func (api *PluginAPI) GetUnsanitizedConfig() *model.Config {
+	return api.app.GetUnsanitizedConfig()
```

Comment:
I think it's ok if you just do `api.app.Config().Clone()` here instead.

Reply:
Hi @ali-farooq0, just passing through. Why call `api.app.Config().Clone()` instead of calling the `GetUnsanitizedConfig()` method?

Reply:
Hey @ides15, since we're not doing anything special for this call other than just returning the cloned config, don't think that warrants another proxy function that's essentially doing that. If in the future we need to perform new operations, we can just update the implementation without affecting plugin clients.

We have other examples where we're doing something similar: https://github.com/mattermost/mattermost-server/pull/11928/files#diff-80d2a5d8acd634624ac04f5c6365a550R90

Reply:
Ah, ok. Thanks!